mlir/include/mlir/Dialect/Rock/Passes.td

```diff
@@ -98,4 +98,8 @@ def RockLowerReducePass : Pass<"rock-lower-reduce", "::mlir::func::FuncOp"> {
   let dependentDialects = ["rock::RockDialect", "func::FuncDialect", "gpu::GPUDialect"];
 }
 
+def RockMakeGepsInboundsPass : Pass<"rock-make-geps-inbounds", "::mlir::LLVM::LLVMFuncOp"> {
```

Comment:
It might be reasonable to make set this value to `""` so this can be run on any root.

Reply:
let  me try

Reply:
done